CONTRIBUTING.md

```diff
@@ -7,29 +7,23 @@ just a few small guidelines you need to follow.
 
 Contributions to this project must be accompanied by a Contributor License
 Agreement. You (or your employer) retain the copyright to your contribution,
-this simply gives us permission to use and redistribute your contributions as
+the CLA simply gives us permission to use and redistribute your contributions as
 part of the project. Head over to <https://cla.developers.google.com/> to see
 your current agreements on file or to sign a new one.
 
 You generally only need to submit a CLA once, so if you've already submitted one
 (even if it was for a different project), you probably don't need to do it
 again.
 
-## Code reviews
```

Comment:
I'm not sure why you deleted this.

Reply:
Hey Adam. Thanks for taking the time to review this PR. I must've removed it by accident/must've thought it's redundant. I don't remember exactly why. I've fixed the issue by adding the heading back to the file.

Reply:
Hey @adammichaelwood Just following up on this

Reply:
Hey @adammichaelwood Let me know if you need help with anything else